adding fuji75 pcb support (#22158)
qmk/qmk_firmware: keyboards/sakura_workshop/fuji75/solder/readme.md

# Fuji75 Solder

![Fuji75 Solder](https://i.imgur.com/tv6YdWt.png)

> 

Reviewer: ```suggestion
```

* Keyboard Maintainer: [Freather](https://github.com/CMMS-Freather)
* Hardware Supported: PCB, Atmega32u4

Make example for this keyboard (after setting up your build environment):

    make sakura_workshop/fuji75/solder:default

Flashing example for this keyboard:

    make sakura_workshop/fuji75/solder:default:flash

> ## Bootloader Atmel-DFU
> 
> For reset instruction, use the physical reset button on the back of the keyboard to enter bootloader mode
> 
> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

Reviewer: ```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

# Fuji75 Solder

> ![Fuji75 Solder](https://i.imgur.com/tv6YdWt.png)

Reviewer 2: ```suggestion
![Fuji75 Solder](https://i.imgur.com/tv6YdWth.png)
```
